engine/template.go

```diff
@@ -119,7 +122,22 @@ func (t *Templates) renderHTMLTemplate(key TemplateKey, breadcrumbs []Breadcrumb
 
 func (t *Templates) renderNonHTMLTemplate(key TemplateKey, params interface{}) {
 	file := filepath.Clean(filepath.Join(key.Directory, key.Name))
-	compiled := texttemplate.Must(texttemplate.New(filepath.Base(file)).Funcs(combinedFuncs).ParseFiles(file))
+	var compiled *texttemplate.Template
+	if _, err := os.Stat(file); os.IsNotExist(err) {
+		gzipFile := file + ".gz"
+		if _, err := os.Stat(gzipFile); os.IsNotExist(err) {
+			log.Fatalf("unable to find (gzipped) file %s", key.Name)
+		} else if err != nil {
+			log.Fatal(err)
+		}
+		fileContents, err := gunzip(gzipFile)
+		if err != nil {
+			log.Fatalf("unable to decompress gzip file %s", gzipFile)
+		}
+		compiled = texttemplate.Must(texttemplate.New(filepath.Base(file)).Funcs(combinedFuncs).Parse(string(fileContents)))
+	} else {
+		compiled = texttemplate.Must(texttemplate.New(filepath.Base(file)).Funcs(combinedFuncs).ParseFiles(file))
```

Comment:
kom je hier nu ooit terecht?

Reply:
Als het goed is wel, als `stijl.json` wel bestaat. Maar zal ik nog even testen